test/boost/chunked_vector_test.cc

```diff
@@ -181,13 +181,12 @@ BOOST_AUTO_TEST_CASE(tests_reserve_partial) {
     auto size_dist = std::uniform_int_distribution<unsigned>(1, 1 << 12);
 
     for (int i = 0; i < 100; ++i) {
-        utils::chunked_vector<uint8_t> v;
-        const auto orig_size = size_dist(rand);
-        auto size = orig_size;
-        while (size) {
-            size = v.reserve_partial(size);
+        utils::chunked_vector<uint8_t, 512> v;
```

Comment:
Won't this fail to compile after the previous commit, which makes `reserve_partial()` return `void`?

Reply:
yes it will fail. I have fixed this in V3.